setup.py

```diff
@@ -76,10 +76,10 @@ def run(self):
     install_requires=[
         'importlib-metadata<4;python_version<"3.8.0"',
         'debugpy>=1.0.0',
```

Comment:
Maybe also pin debugpy?
```suggestion
        'debugpy>=1.0.0,<2.0',
```